src/data.js

```diff
@@ -1,5 +1,26 @@
 // keep it commonjs export
 module.exports = [
+    {
+    name: 'Jesse Stilwell',
+    description: 'Hobbyist full stack dev, twenty-first century digital boy, music maker, and professional geek.',
+    url: 'https://stilwell.io/uses',
+    twitter: '@owlbynight',
+    emoji: '🦉',
+    country: 'US',
```

Comment:
To make the CI pass you'll need to swap this with the US flag emoji 😄 

We should really support both